.github/workflows/deploy_branch_preview.yml

```diff
@@ -60,8 +60,21 @@ jobs:
             ${{ runner.os }}-
       - name: Install Packages
         run: npm install
+      - name: Get Latest Release
+        id: latestRelease
+        uses: pozetroninc/github-action-get-latest-release@master
+        with:
+          repository: ${{ github.repository }}
```

Comment:
Is it possible to get the latest tag by walking up the tree of the target branch instead? I doubt a global check of the latest release is scalable when we have multiple branches